src/dmd/dsymbolsem.d

```diff
@@ -1616,6 +1616,30 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
     {
         // Should be merged with PragmaStatement
         //printf("\tPragmaDeclaration::semantic '%s'\n", pd.toChars());
+        if (global.params.mscoff)
+        {
+            if (pd.ident == Id.linkerdirective)
+            {
+                if (!pd.args || pd.args.dim != 1)
+                    pd.error("string expected for linker directive");
```

Comment:
`one string argument expected for pragma(linkerDirective)`

Reply:
Like this: https://github.com/dlang/dmd/blob/master/src/dmd/dsymbolsem.d#L1658 ?